src/Core/Style.js

```diff
@@ -17,7 +17,7 @@ const inv255 = 1 / 255;
 const canvas = (typeof document !== 'undefined') ? document.createElement('canvas') : {};
 
 function baseAltitudeDefault(properties, ctx) {
-    return ctx?.coordinates?.z || ctx?.collection?.center?.z || 0;
+    return ctx?.coordinates?.z || 0;
```

Comment:
should we default it to the terrain elevation using demutils?

Reply:
What you suggest is, in case there is not elevation value available, we should set the altitude to the terrain level and then in not to zero ? like :
return ctx?.coordinates?.z || itowns.DEMUtils.getTerrainObjectAt(view.tileLayer, ctx.coordinates, 0) || 0;

Reply:
Yep that's the idea but the code would be `return ctx?.coordinates?.z || itowns.DEMUtils.getElevationValueAt(view.tileLayer, ctx.coordinates, 0) || 0;` 

We would need to have knowledge of the `view` here though... Moreover it can be costly (to evaluate?) and not what the user wants (even if I don't see any usecase where the user would not like to have its data projected on its terrain by default if no other base altitude is provided). It should not block this PR and we can continue the discussion elsewhere but

 in some cases that I don't foresee yet though. BUT it seems more adapted functionally to me However, it  functionally it would allow to project the data on the terrain automatically instead of having to provide a method to

Reply:
@jailln Since the goal of the previous PR was to have a proper separation of concerns in the `Style` module, I don't think this is a good idea to re-introduce a dependency to the `View`. I thought that the elevation was relative to the ellipsoid?